q2_diversity/_ordination.py

```diff
@@ -10,8 +10,20 @@
 import pandas as pd
 
 
-def pcoa(distance_matrix: skbio.DistanceMatrix) -> skbio.OrdinationResults:
-    return skbio.stats.ordination.pcoa(distance_matrix)
+def pcoa(distance_matrix: skbio.DistanceMatrix,
+         number_of_dimensions: int = 0) -> skbio.OrdinationResults:
```

Comment:
I did like @ElDeveloper's initial idea of using `None` instead of `0` here. Since it would mean *any* value passed to `number_of_dimensions` results in fsvd. Then the range can be `Range(1, None)`.

Reply:
I would be OK with that since there are no contracts or requirements that there's a 1:1 match between interfaces.